Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/client/transport.go

```diff
@@ -117,6 +118,25 @@ func TLSConfigFor(config *Config) (*tls.Config, error) {
 	return tlsConfig, nil
 }
 
+// tlsConfigKey returns a unique key for tls.Config objects returned from TLSConfigFor
```

Comment:
TLSConfigFor doesn't use them. If it started to, this would need to incorporate them into the key